packages/@coorpacademy-components/src/molecule/card-content/index.js

```diff
@@ -154,13 +165,14 @@ const ContentInfo = ({
     <div
       data-name="info"
       className={classnames(
+        THEMES[theme],
         style.infoWrapper,
         mode === MODES.HERO ? style.hero : style.card,
         disabled ? style.progressBarDisabled : null,
         externalContent ? style.externalContent : null
       )}
     >
-      <ContentTypeInfo mode={mode} type={type} externalContent={externalContent} />
+      <ContentTypeInfo mode={mode} type={type} externalContent={externalContent} theme={theme} />
```

Comment:
```suggestion
      <ContentTypeInfo mode={mode} type={type} externalContent={externalContent}/>
```
pas besoin de theme l'élément root a déjà la class coorpmanager